api/elastic.py

```diff
@@ -60,16 +61,20 @@ def generateSearchQuery(self, params):
                 searchClauses.append(Q('bool', should=[
                     self.titleQuery(escapedQuery),
                     self.authorQuery(escapedQuery),
-                    self.subjectQuery(escapedQuery)
+                    self.subjectQuery(escapedQuery),
+                    self.identifierQuery(escapedQuery),
+                    self.authorityQuery(escapedQuery)
                 ]))
             elif field == 'title':
                 searchClauses.append(self.titleQuery(escapedQuery))
             elif field == 'author':
                 searchClauses.append(self.authorQuery(escapedQuery))
             elif field == 'subject':
                 searchClauses.append(self.subjectQuery(escapedQuery))
-            elif field == 'viaf' or field == 'lcnaf':
+            elif field == 'viaf' or field == 'lcnaf' or field in authorityList:
```

Comment:
If we want to do this we need to extend the `authorityQuery` method, which I'm not opposed to, but might be unnecessary depending on how we implement the identifier query (see comment below)